src/server/req_quejob.c

```diff
@@ -2571,6 +2571,10 @@ locate_new_job(struct batch_request *preq, char *jobid)
 void
 req_resvSub(struct batch_request *preq)
 {
+	/*
+	 * buf and buf1 are used to hold user@hostname stings together
+	 * with a small amount (less than 64 characters) of text.
+	 */
 	char		 buf[PBS_MAXUSER + PBS_MAXHOSTNAME + 64];
 	char		 buf1[PBS_MAXUSER + PBS_MAXHOSTNAME + 64] = {0};
```

Comment:
Should this be = {'\0'}?

Reply:
They are equivalent for all intents and purposes. If I change that one, then I need to change 5 more in that function. I'll leave it as is.